workspaces/diff-engine/src/queries/endpoint.rs

```diff
@@ -6,6 +7,8 @@ use crate::state::endpoint::{
 use crate::HttpInteraction;
 use petgraph::graph::Graph;
 use petgraph::visit::EdgeFilteredNeighborsDirected;
+use serde_json::json;
```

Comment:
If this is only used for tests, it should be imported inside the `test` module. Wouldn't want anyone to think json as any other place in the actual implementation of `EndpointQueries` 😅

Reply:
- [ ] move test dependency into test module